src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
it's a waste to do this for existing users:
- if the user didn't change their preferredName, you're making an unnecessary call
- if the user _did_ change their preferredName, you're not capturing that in `updateUserProfile`